src/utils.js

```diff
@@ -27,6 +27,21 @@ class Utils {
     return newName;
   }
 
+  resolvePath(...values) {
+    for (let i = 0; i < values.length; i++) {
+      let value = values[i];
+
+      if (value) {
+        if (value.path) {
+          value = value.path;
+        }
+
+        return value;
+      }
+    }
+    return false;
```

Comment:
Maybe `return '';` instead? Otherwise `encodeURIComponent(redirectTo)` will evaluate to `'false'`. Or change `encodeURIComponent(redirectTo)` to `encodeURIComponent(redirectTo || '')`.